### PR TITLE
Fixing unwanted newline in date block

### PR DESCRIPTION
**Pull request by felipeparaujo**

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Comment by tatitati** [comment]
simple 👍

File: app/Resources/views/schedules/by_base.html.twig [hunk 1]
```diff
@@ -1,8 +1,6 @@
 {% extends 'base_ds2013.html.twig' %}
 
-{%- block title -%}
-    {{ meta_context.titlePrefix }} - {{ tr('schedules') }}, {{ block('date') }}
-{%- endblock -%}
+{% block title %}{{ meta_context.titlePrefix }} - {{ tr('schedules') }}, {{ block('date') }}{% endblock %}
```
Comment (votemike) [inline app/Resources/views/schedules/by_base.html.twig]: Did `{%-` not make it spaceless?

Comment (felipeparaujo) [inline app/Resources/views/schedules/by_base.html.twig]: yeah it did, but this block is so short we could have it in one line, at which point we don't need the `{%-` anymore

**Comment by BjornTwachtmann** [comment]
looks sensible

**Pull request merged**